sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/organisms/InformationMessageCardViewScreen.kt

```diff
@@ -0,0 +1,91 @@
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ExamplesSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.PlaceholderSlot
+import uk.gov.hmrc.sample_compose_fragments.presentation.screens.sampletemplate.ScreenScrollViewColumn
+
+@Composable
+fun InformationMessageCardViewScreen() {
+    ScreenScrollViewColumn {
+        PlaceholderSlot {
+            InformationMessageCardView(
+                headline = stringResource(id = R.string.info_message_placeholder_headline),
+                text = stringResource(id = R.string.info_message_placeholder_headline),
+                messageType = MessageType.URGENT,
+                buttons = listOf(InformationMessageButton("Headline button", MessageType.URGENT, onClick = {}))
+            )
+        }
+
+        ExamplesSlot {
+            // Example 1
+                InformationMessageCardView(
+                    headline = stringResource(id = R.string.info_message_example_1_headline),
+                    messageType = MessageType.INFO
+                )
+
+            Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing16))
+
+            // Example 2
+                InformationMessageCardView(
+                    headline = stringResource(id = R.string.info_message_example_2_headline),
+                    messageType = MessageType.WARNING,
+                    buttons = listOf(
+                        InformationMessageButton("Do something", MessageType.WARNING, onClick = {}),
+                        InformationMessageButton("Do something", MessageType.WARNING, ButtonType.OUTLINE, onClick = {})
+                    )
+                )
+
+            Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing16))
+
+            // Example 3
+                InformationMessageCardView(
+                    headline = stringResource(id = R.string.info_message_example_3_headline),
+                    text = stringResource(id = R.string.info_message_example_3_body),
+                    headlineContentDescription = stringResource(id = R.string.info_message_example_3_headline_content_description),
+                    messageType = MessageType.INFO,
+                    buttons = listOf(
+                        InformationMessageButton("Do something", MessageType.INFO, onClick = {}),
+                        InformationMessageButton("Do something", MessageType.INFO, ButtonType.OUTLINE, onClick = {})
+                    )
+                )
+
+            Spacer(modifier = Modifier.height(HmrcTheme.dimensions.hmrcSpacing16))
+
+            // Example 4
+                InformationMessageCardView(
+                    headline = stringResource(id = R.string.info_message_example_4_headline),
+                    headlineContentDescription = stringResource(id = R.string.info_message_example_4_headline_content_description),
+                    messageType = MessageType.NOTICE
+                )
+        }
```

Comment:
I think you forgot to update the indentation after removing the cards here.. can we tidy it up a bit and re-indent?